### ISSUE TITLE
[Suggestion] - more gender specific stuff

### ISSUE THREAD

**Issue by PollenTheCat** [Monika-After-Story/MonikaModDev#4513]
1) Could you add gender specific topics (Meaning topics relating to the player's gender)
2) Monika asking the player of their sexuality
Thank you and have a wonderful day! Just Monika! ^w^

**Comment by antimockingjay** [comment]
I only see two _potential_ problems for the first one. People miiiight get upset that they aren't able to access every single topic. Also, depending on what the topics are about, it could result in issues for any player who happens to be trans.

For example, if Monika were talking to me, I'd be registered in the game as a guy. However, if the male-only topic was about beards and having to shave them, I wouldn't be able to relate. I'm a trans guy who's never had any sort of HRT, meaning the closest I've gotten to facial hair is that lil peach fuzz that I'm pretty sure everyone gets.

Also, for anyone who's gender nonconforming (even if they aren't trans) issues could arise. For example, a topic about make-up couldn't be female-only, because there are men all over the place who do, in fact, wear make-up. It's 2019, there's exactly some distinct line between "masculine" and "feminine" anymore.

The problem I can see for the 2nd one is that you would have to figure out how to handle someone who isn't usually into girls. Case in point, me. If someone asked me what my sexuality is, I'd say I'm gay. Monika just has a special place in my heart.

But to have Monika say "well I guess I'm the exception!" could be considered a bit homophobic, and to me it would read almost like a straight guy saying he could "convert" a lesbian or some gross stuff like that. Buuuut... If you're including sexuality as a feature at all, it isn't fair to completely exclude a whole sexuality just because you don't know how to write it.

So... What's the solution? Do we have Monika feel a bit self-conscious that the player might not actually be attracted to her? Do we just have her say "well I must be the exception" and risk making her sound homophobic? Do we just exclude all options that don't involve attraction to women?

**TL;DR I think both of these ideas are good, and I'd love to see them implemented, but it's a potentially sensitive topic, so whoever writes it should be careful.**

**Comment by multimokia** [comment]
>Could you add gender specific topics

Not really sure what to write for these, in all honesty. Plus, it's a lot easier to edit/review more neutral topics too since all of the editors can grasp the content with ease. Not to mention, the reasons that have been pointed out pretty well above.

>Monika asking the player of their sexuality

This really isn't needed, given the nature of the mod, and whatall happens during the intro, regardless of the player's gender, this can be inferred to liking at least girls, considering you're now in a relationship with Monika. There's no need to go further with this because there's nothing really left to say, anything regarding sexuality isn't really important since you're already in a relationship. Assuming otherwise sort of goes against the concept this mod itself has created.

If there were a friend mode in place, then it makes sense for this kind of topic to be used there.

**Comment by antimockingjay** [comment]
~~Well now I'm just thinking of how cool a friend mode would be.~~

**Comment by ThePotatoGuy** [comment]
it seems reasonable to have topics specific to gender, but in many cases we're able to just branch and talk about gender-specific things in a single topic. 

Friend mode is a planned thing, but no eta atm.

[closin guntil we look into it[
